sql-update/make-yoder.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE actors
+  SET "firstName" = 'Baby',
+      "lastName"  = 'Yoda'
+where "actorId" = 15;
+
+SELECT * from actors where "actorId" = 15;
```

Comment:
It's best to use the `returning` syntax since it's more efficient and guaranteed to return the proper value.

`returning` is proprietary to postressql, but almost all databases will have a way to accomplish the same thing.

However, for updates, it's not always useful to return anything.